app/datarequest/datamanagerreview.js

```diff
@@ -4,51 +4,67 @@ import { render } from "react-dom";
-                        prschemaresponse, prdataresponse) => {
-        datarequestFormData = dataresponse.data;
-        datarequestSchema   = schemaresponse.data.schema;
-        datarequestUiSchema = schemaresponse.data.uiSchema;
-        prFormData          = prdataresponse.data;
-        prSchema            = prschemaresponse.data.schema;
-        prUiSchema          = prschemaresponse.data.uiSchema;
-
+$(document).ready(function() {
+
+    var datarequestSchema   = {};
+    var datarequestUiSchema = {};
+    var datarequestFormData = {};
+
+    // Get data request
+    Yoda.call('datarequest_get',
+        {request_id: requestId},
+        {errorPrefix: "Could not get datarequest"})
+    .then((datarequest) => {
+        datarequestFormData = JSON.parse(datarequest.requestJSON);
+    })
+    // Get data request schema and uiSchema
+    .then(async function() {
+        let schema = await axios.get("/datarequest/datarequest/schema");
+        datarequestSchema   = schema.data.schema;
+        datarequestUiSchema = schema.data.uiSchema;
+    })
+    // Render data request as disabled form
+    .then(() => {
         render(<ContainerReadonly schema={datarequestSchema}
                                   uiSchema={datarequestUiSchema}
                                   formData={datarequestFormData} />,
-            document.getElementById("datarequest")
+               document.getElementById("datarequest")
         );
+    });
 
+    var prSchema   = {};
+    var prUiSchema = {};
+    var prFormData = {};
+
+    // Get preliminary review
+    Yoda.call('datarequest_preliminary_review_get',
+        {request_id: requestId},
+        {errorPrefix: "Could not get preliminary review"})
+    .then((preliminary_review) => {
+        prFormData = JSON.parse(preliminary_review);
+    })
+    // Get preliminary review schema and uiSchema
+    .then(async function() {
+        let schema = await axios.get("/datarequest/datarequest/preliminaryReviewSchema");
+        prSchema   = schema.data.schema;
+        prUiSchema = schema.data.uiSchema;
+    })
+    // Render preliminary review as disabled form
+    .then(() => {
         render(<ContainerReadonly schema={prSchema}
                                   uiSchema={prUiSchema}
                                   formData={prFormData} />,
             document.getElementById("preliminaryReview")
         );
-    }));
+    });
 
-var datamanagerReviewSchema = {};
-var datamanagerReviewUiSchema = {};
-var form = document.getElementById('datamanagerReview');
+    var datamanagerReviewSchema = {};
+    var datamanagerReviewUiSchema = {};
+    var form = document.getElementById('datamanagerReview');
 
-// Get the schema of the data request review form for the data manager
-axios.get("/datarequest/datarequest/datamanagerReviewSchema")
+    // Get the schema of the data request review form for the data manager
+    axios.get("/datarequest/datarequest/datamanagerReviewSchema")
```

Comment:
Same change as in app/datarequest/preliminaryreview.js, use fetch instead of axios.